packages/relay-kit/src/packs/safe-4337/utils.ts

```diff
@@ -13,6 +13,7 @@ import {
 } from '@safe-global/protocol-kit'
 import { ethers } from 'ethers'
 import { EIP712_SAFE_OPERATION_TYPE, INTERFACES } from './constants'
+import { SAFE_WEBAUTHN_SHARED_SIGNER_ADDRESS } from './Safe4337Pack'
```

Comment:
This generates a "Circular dependency" between files, should be added to `constants`

Reply:
done!